refactor: Update `README.md` and improved repository (#926)
parse-community/Parse-SDK-Flutter: packages/flutter/README.md

<img src="https://user-images.githubusercontent.com/5673677/166121333-2a144ce3-95bc-45d6-8840-d5b2885f2046.png" alt="parse-repository-header-sdk-flutter" style="max-width: 100%;">

---

> [![pub package](https://img.shields.io/pub/v/parse_server_sdk_flutter.svg)](https://pub.dev/packages/parse_server_sdk_flutter) [![Pub Likes](https://img.shields.io/pub/likes/parse_server_sdk_flutter)](https://pub.dev/packages/parse_server_sdk_flutter/score)  [![Pub Popularity](https://img.shields.io/pub/popularity/parse_server_sdk_flutter)](https://pub.dev/packages/parse_server_sdk_flutter/score) [![Pub Points](https://img.shields.io/pub/points/parse_server_sdk_flutter)](https://pub.dev/packages/parse_server_sdk_flutter/score)

Author: ok

---

- [Compatibility](#compatibility)

> - [Getting Started](#getting-started)

Reviewer: ```suggestion
- [Getting Started](#getting-started)
- [Documentation](#documentation)
- [Contributing](#contributing)
```

| Flutter 3.3  | Jan 2024       | ✅ Yes                                         |
| Flutter 3.0  | Jul 2023       | ❌ No (Parse Flutter SDK requires Flutter >=3.3.0) |

## Getting Started

> To install, either add [dependency in your pubspec.yaml file](https://pub.dev/packages/parse_server_sdk_flutter/install).

Reviewer: ```suggestion
To install, add the Parse Flutter SDK as a [dependency](https://pub.dev/packages/parse_server_sdk_flutter/install) in your `pubspec.yaml` file.
```

> See the SDK Usage [guide][guide].

Reviewer: ```suggestion
## Documentation

Find the full documentation in the [Parse Flutter SDK guide][guide].
```

> # Contributing

Reviewer: ```suggestion
## Contributing
```
